# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1205954982 14400
#      Wed Mar 19 15:29:42 2008 -0400
# Node ID d6d1ee7717d032921608b88522bf1cbb710459bf
# Parent  704b7a1098d0914143e9fe05efbd9f5632e7459c
micro patch of indexing bug

diff --git a/test/test_index-wfi-f.m b/test/test_index-wfi-f.m
--- a/test/test_index-wfi-f.m
+++ b/test/test_index-wfi-f.m
@@ -400,16 +400,17 @@
 %! a9(:,:,1,2) = [1, 2; 3, 4];
 %! a10(:,:,1,1) = [1, 2; 3, 4];
 %! a10(:,:,2,1) = [1, 2; 3, 4];
 %! a10(:,:,1,2) = [1, 2; 3, 4];
 %! a10(:,:,2,2) = [1, 2; 3, 4];
 %! a11 = zeros (1, 0);
 %! a12 = zeros (1, 0, 2);
 %! a13 = zeros (1, 1, 0);
+%! clear a14; a14(1:10,1) = 1:10;
 %! b1 = [1; 3; 2; 4; 5; 7; 6; 8];
 %! b2 = [1, 2, 5, 6; 3, 4, 7, 8];
 %! b3 = [1; 3];
 %! b4(:,:,1) = [1, 2; 3, 4];
 %! b4(:,:,2) = [5, 6; 7, 8];
 %! b5 = [1; 3];
 %! b6(:,:,1,1) = [1; 3];
 %! b6(:,:,1,2) = [1; 3];
@@ -429,16 +430,17 @@
 %! assert(a(1:2,1), a6);
 %! assert(a(:,:,1), a7);
 %! assert(a(:,:,[1,1]), a8);
 %! assert(a(:,:,1,[1,1]), a9);
 %! assert(a(:,:,[1,1],[1,1]), a10);
 %! assert(a(1,[]), a11);
 %! assert(a(1,[],[1,1]), a12);
 %! assert(a(1,1,[]), a13);
+%! assert(a14, (1:10)');
 %! assert(b(:), b1);
 %! assert(b(:,:), b2);
 %! assert(b(:,1), b3);
 %! assert(b(:,:,:), b4);
 %! assert(b(:,1,1), b5);
 %! assert(b(:,1,1,[1,1]), b6);
 %! assert(b(1,3), b7);
 %! assert(b(1,[3,4]), b8);
